io.github.cges30901.hmtimer.appdata.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Since you are upstream this does go upstream. Same for the desktop file and icons.